src/components/ItemDetail/ItemDetailMetadata.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LineEntry from './LineEntry';
+
+const ItemDetailMetadata = props => {
+  if (!props.item) {
+    return [];
+  }
+
+  const {
+    title_tesim,
+    admin_set_tesim,
+    subject_topical_label_tesim,
+    subject_topical_tesim,
+    genre_label_tesim,
+    genre_tesim,
+    human_readable_type_tesim,
+    ark_tesim,
+    rights_statement_tesim
+  } = props.item;
+
+  return (
+    <section className="item-section contain-970 item-categories-wrapper">
+      <div id="tab-container">
```

Comment:
At some point, there could be an argument to create a "Tab" component, which accepts it's tab type, then renders out the `LineEntry` components.... in future work.